open-api/rest-catalog-open-api.yaml

```diff
@@ -4647,8 +4647,7 @@ components:
 
     ServiceUnavailableResponse:
       description:
-        The service is not ready to handle the request. The client should wait and retry.
-
+        The service is not ready to handle the request, request could have been partially processed.
```

Comment:
Should we clarify that it should be retried only if `RETRY_AFTER` header exists in the response?